### ISSUE TITLE
pyparsing, why is it needed?

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/homebrew-tap#38]
There is no `pyparsing` when I do a normal install via pip - why does the brew formula have pyparsing in it?

```
tw@MacBook-Pro ~ % mkdir borg14-exp
tw@MacBook-Pro ~ % cd borg14-exp 
tw@MacBook-Pro borg14-exp % python3 -m venv borg-env
tw@MacBook-Pro borg14-exp % . borg-env/bin/activate

(borg-env) tw@MacBook-Pro borg14-exp % pip install borgbackup
Collecting borgbackup
  Using cached borgbackup-1.4.0.tar.gz (3.8 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting msgpack<=1.1.0,>=1.0.3 (from borgbackup)
  Using cached msgpack-1.1.0-cp313-cp313-macosx_11_0_arm64.whl.metadata (8.4 kB)
Collecting packaging (from borgbackup)
  Using cached packaging-24.2-py3-none-any.whl.metadata (3.2 kB)
Using cached msgpack-1.1.0-cp313-cp313-macosx_11_0_arm64.whl (81 kB)
Using cached packaging-24.2-py3-none-any.whl (65 kB)
Building wheels for collected packages: borgbackup
  Building wheel for borgbackup (pyproject.toml) ... done
  Created wheel for borgbackup: filename=borgbackup-1.4.0-cp313-cp313-macosx_15_0_arm64.whl size=911156 sha256=0796db3f4ae0f054a9ed8ddeebfe7d7df86b39439da57e82278c44fae4d76e10
  Stored in directory: /Users/tw/Library/Caches/pip/wheels/66/12/f9/6a17239116ee0b14470a895e90a5083dd86e00661e5197416a
Successfully built borgbackup
Installing collected packages: packaging, msgpack, borgbackup
Successfully installed borgbackup-1.4.0 msgpack-1.1.0 packaging-24.2

(borg-env) tw@MacBook-Pro borg14-exp % borg -V
borg 1.4.0

(borg-env) tw@MacBook-Pro borg14-exp % pip freeze
borgbackup==1.4.0
msgpack==1.1.0
packaging==24.2
```

**Comment by ThomasWaldmann** [comment]
Forgot llfuse, thus repeated with that:

```
(borg-env) tw@MacBook-Pro borg14-exp % pip uninstall borgbackup      
Found existing installation: borgbackup 1.4.0
Uninstalling borgbackup-1.4.0:
  Would remove:
    /Users/tw/borg14-exp/borg-env/bin/borg
    /Users/tw/borg14-exp/borg-env/bin/borgfs
    /Users/tw/borg14-exp/borg-env/lib/python3.13/site-packages/borg/*
    /Users/tw/borg14-exp/borg-env/lib/python3.13/site-packages/borgbackup-1.4.0.dist-info/*
Proceed (Y/n)? y
  Successfully uninstalled borgbackup-1.4.0

(borg-env) tw@MacBook-Pro borg14-exp % pip install 'borgbackup[llfuse]'
Collecting borgbackup[llfuse]
  Using cached borgbackup-1.4.0-cp313-cp313-macosx_15_0_arm64.whl
Requirement already satisfied: msgpack<=1.1.0,>=1.0.3 in ./borg-env/lib/python3.13/site-packages (from borgbackup[llfuse]) (1.1.0)
Requirement already satisfied: packaging in ./borg-env/lib/python3.13/site-packages (from borgbackup[llfuse]) (24.2)
Collecting llfuse>=1.3.8 (from borgbackup[llfuse])
  Using cached llfuse-1.5.1-cp313-cp313-macosx_15_0_arm64.whl
Installing collected packages: llfuse, borgbackup
Successfully installed borgbackup-1.4.0 llfuse-1.5.1

(borg-env) tw@MacBook-Pro borg14-exp % rehash                          
(borg-env) tw@MacBook-Pro borg14-exp % borg -V
borg 1.4.0

(borg-env) tw@MacBook-Pro borg14-exp % pip freeze
borgbackup==1.4.0
llfuse==1.5.1
msgpack==1.1.0
packaging==24.2
```

**Comment by fasmat** [comment]
Should be fixed by #39
